test/test_cpp_extensions_jit.py

```diff
@@ -327,6 +327,53 @@ def test_inline_jit_compile_extension_cuda(self):
         z = module.cos_add(x, y)
         self.assertEqual(z, x.cos() + y.cos())
 
+    @unittest.skipIf(not TEST_CUDA, "CUDA not found")
```

Comment:
Should we add the `not (TEST_CUDA or TEST_ROCM)` in this PR, or do you want to create a follow-up PR to enable many of these `TEST_CUDA`-only tests?

Reply:
Also I should note that the test script where you've added this test is one of the "slow" ones that does not run automatically in CI.

Reply:
Let's enable them all. Not at all or not run on all instances?

Reply:
Next, I'll try a non-stupid variant...